src/Assistant.ts

```diff
@@ -345,11 +346,12 @@ function createSetSuggestedPrompts(args: AllAssistantMiddlewareArgs): SetSuggest
   const { channelId: channel_id, threadTs: thread_ts } = extractThreadInfo(payload);
 
   return (params: Parameters<SetSuggestedPromptsFn>[0]) => {
-    const { prompts } = params;
+    const { prompts, title } = params;
     return client.assistant.threads.setSuggestedPrompts({
       channel_id,
       thread_ts,
       prompts,
+      ...(title && { title }),
```

Comment:
It seems simply passing `title,` here works too. Is there any particular reason to have this more complex one?

Reply:
I think to try to eliminate potentially `undefined` values? `title` is optional, so could be undefined. However, the API client eliminates `undefined` values from the final HTTP request issued (I just tested this), so indeed we could eliminate the truthiness check.

```suggestion
      title,
```

Reply:
Yep, this was to only send the key along if it was provided by the user. Will remove.